code/obj/window.dm

```diff
@@ -987,17 +987,27 @@
 
 
 	auto
-		name = "reinforced autowindow grille spawner"
+		name = "autowindow grille spawner (will apply in time)"
 		win_path = "/obj/window/auto/reinforced"
```

Comment:
the fact that it treats paths as strings is kinda disgusting ngl

Reply:
Perhaps you can change that in a pr.